File: Python 3/1010.py
c1, n1, v1 = (input().split(' '))
c2, n2, v2 = (input().split(' '))

v = float(int(n1) * float(v1) + int(n2) * float(v2))

print('VALOR A PAGAR: R$ {:.2f}'.format(v))

File: C++/1010.cpp
#include <iostream>
 
using namespace std;
 
int main() {
    float total;

    for (int i = 0; i <= 1; i ++){
        int c, n;
        float v;
        cin >> c >> n >> v;
        total = total + (n * v);
    }
    cout << fixed;
    cout.precision(2);
    cout << "VALOR A PAGAR: R$ " << total << endl;
    
    return 0;
}